src/main/java/database/ChatInterface.java

```diff
@@ -0,0 +1,21 @@
+package database;
+
+import java.util.List;
+
+import models.Message;
+
+public interface ChatInterface {
```

Comment:
This interface shouldn't be named `ChatInterface`. As we agreed it should be [Functionality] + Handler (e.g. `ChatHandler`).